src/saturn_engine/worker_manager/api/topologies.py

```diff
@@ -0,0 +1,18 @@
+from flask import Blueprint
+
+from saturn_engine.database import session_scope
+from saturn_engine.stores import topologies_store
+from saturn_engine.utils.declarative_config import BaseObject
+from saturn_engine.utils.flask import Json
+from saturn_engine.utils.flask import jsonify
+from saturn_engine.utils.flask import marshall_request
+
+bp = Blueprint("topologies", __name__, url_prefix="/api/topologies")
+
+
+@bp.route("", methods=("PUT",))
```

Comment:
route -> `patch`? `/api/topologies` could be used one day to actually create components.

Reply:
`PUT /api/topologies/patch`